api.go

```diff
@@ -31,6 +31,12 @@ func CreatePod(podConfig PodConfig) (*Pod, error) {
 		return nil, err
 	}
 
+	// Create the network.
+	err = p.network.add(p)
```

Comment:
Some more details: We really want the networking namespace to be created and ready before we create the pod. That has several implications:

1. `network.add()` should not take a pod as a parameter, but a network config and an ID.
2. `createPod()` should take the list of created networking pairs as an additional parameter.